pytorch_blade/src/compiler/mlir/converters/mhlo_conversion.cpp

```diff
@@ -207,7 +209,12 @@ bool IsMlirMhloSupported(const torch::jit::Node& node) {
     if (!node.kind().is_prim() && !AllTensorTypeAnalyzed(node)) {
       return false;
     }
-    return IsTorchMlirSupported(node);
+    auto supported = IsTorchMlirSupported(node);
+    if (!(supported || prim::FusionGroup == node.kind() ||
+          prim::Param == node.kind())) {
+      LOG(WARNING) << "Not in white list: " << node << std::endl;
```

Comment:
warning level always print the logs, can we use `GRAPH_DEBUG` which is easy to enable or not.

Reply:
done, only do logging when `TORCH_BLADE_MHLO_DEBUG_LOG` is open